### PR TITLE
fix(dependencies): Update and fix dependencies

### LINKED ISSUE #272: Add types as true dependencies
Authmosphere tooling relies on external types e.g. from express. These are needed in type relying environments like typescript. As some tooling not relying on express might be used it is not cumbersome to rely on an external import of types (e.g. as peer dependency). Therefor the types must be part of the release package and therefor a true dependency

### PR DESCRIPTION
**Pull request by Retro64**
close #272

### CODE REVIEW

**Comment by Retro64** [comment]
:+1:

File: package.json [hunk 1]
```diff
@@ -25,38 +25,38 @@
     "node": "^12.22.0 || ^14.17.0 || >=16.0.0"
   },
   "dependencies": {
+"@types/express": "4.17.13",
```
Comment (bzums) [inline package.json:28]: Indent @Retro64

**Review by bzums — COMMENTED** [review]
@Retro64 are we sure that we only import types from express? Otherwise we would also need to move `express` itself from dev dependencies to dependencies.

**Comment by Retro64** [comment]
@bzums I double checked - we use express code in one integration test only. The other imports are only importing interfaces, so the types should be the only hard dependency. I also checked other imports, but I did not see any other type, that needs to be moved to the dependencies

**Comment by Retro64** [comment]
:+1:

**Comment by Retro64** [comment]
:+1:

**Review by Retro64 — COMMENTED** [review]
:+1:

File: src/utils.ts [hunk 2]
```diff
@@ -14,19 +15,7 @@ import {
...
-        reject(error);
-        return;
-      }
-      resolve(data.toString());
-    });
-  });
-
-  return readPromise;
-};
+const fsReadFile = promisify(readFile);
```
Comment (bzums) [inline src/utils.ts:18]: Looks like we can use as "promised" version from node directly here: https://remarkablemark.org/blog/2020/12/20/nodejs-fs-readfile-promise/

Maybe wen can get rid of the `promisify` in that case in general?

Comment (Retro64) [reply src/utils.ts:18]: This is not compatible with node 12 - see last commit

**Comment by bzums** [comment]
👍

**Comment by Retro64** [comment]
:+1:

**Pull request merged**